web/pubspec.yaml

```diff
@@ -0,0 +1,8 @@
+name: samples
```

Comment:
Likewise, the web directory also isn't a valid Flutter project. Please delete

Reply:
It does? I'd prefer to roll a minimal project in the base directory to contain the configuration file rather than this approach. That way at least we won't break CI.

Reply:
Indeed 
<img width="750" alt="image" src="https://user-images.githubusercontent.com/6655696/218704523-c41ccd99-fc1a-4d74-a5b2-5b39a35e512e.png">

What do you mean by minimal project, isn't a pubspec the most minimal you can get?